### DISCUSSION TITLE
Android ipv6 dns.

### DISCUSSION THREAD

**Discussion by Pushergene** [AdguardTeam/AdGuardHome#3669] [Q&A]
Hello,
I am running the latest Adguard Home version on Raspberry Pi OS (latest). The issue: on Android i am only able to change ipv4 dns, not ipv6. Problem: Ads arent getting blocked because i cant disable ipv6 in my Router. Tested on Redmi Note 9 (XIAOMI). I would like to know if that is usual and how to fix it. I am not using Adguard Home as DHCP Server.

**Comment by ameshkov** [comment]
Usually, it's possible to disable IPv6.

Also, you can make AGH to listen to 0.0.0.0, this way it will also has an IPv6 address which you can use in the router settings.

**Reply by Pushergene** [reply]
What do you mean with 0.0.0.0? I mean, my Router does not allow me to disable ipv6 or change dns. So i tried changing manually DNS  on devices, windows has the option for disabling or customize the ipv6dns. But android does not. When i go to dns-leak.com, it shows my isp provider one time, the rest are the upstream dns servers. Im wondering why it seems like that i am the only person with that issue.

**Reply by ameshkov** [reply]
`0.0.0.0` means that AGH would listen to all network interfaces.

> Router does not allow me to disable ipv6 or change dns

That's rather unusual. Can you use AGH's DHCP server instead?
